### PR TITLE
TECH-1332 finish list generation

### PR DESCRIPTION
**Pull request by ericbroder**

### CODE REVIEW

File: organizing_hub/tasks.py [hunk 1]
```diff
@@ -120,60 +75,73 @@ def sync_contact_list_with_bsd_constituents(
...
+                            'first_name': constituent.find('firstname').text,
+                            'last_name': constituent.find('lastname').text,
+                            'point': constituent_point,
+                        },
+                    )
+                    contact_list.contacts.add(contact)
+
+    """Get list limit from max contacts and point."""
+    if point is None or max_contacts is None or max_contacts == 0:
+        list_limit = None
```
Comment (cjmabry) [inline organizing_hub/tasks.py]: We should never have no list_limit. Should enter a default if not empty so we avoid sending accidental massive sends.

File: ourrevolution/settings.py [hunk 2]
```diff
@@ -51,6 +64,14 @@
     'EVENTS_PROMOTE_MAX',
     4000
 ))
+EVENTS_PROMOTE_MAX_DISTANCE_MILES = int(os.environ.get(
+    'EVENTS_PROMOTE_MAX_DISTANCE_MILES',
+    100
```
Comment (cjmabry) [inline ourrevolution/settings.py]: We should probably set this config to something much smaller, but this default is fine.

File: organizing_hub/tasks.py (hunk 1 above)
Comment (ericbroder) [reply organizing_hub/tasks.py]: will change

File: ourrevolution/settings.py (hunk 2 above)
Comment (ericbroder) [reply ourrevolution/settings.py]: seems ok to me? its going to sort by distance anyway if theres too many.

Comment (cjmabry) [reply ourrevolution/settings.py]: I don't see many scenarios where people will be driving 100 miles to an event, and sending them an email for an event they'd never attend only risks an unsubscribe. We can talk about an appropriate number.

**Pull request merged**